src/app/pages/HomePage/Graph/NetworkSelector/index.tsx

```diff
@@ -1,100 +1,133 @@
-import RemoveIcon from '@mui/icons-material/Remove'
 import Typography from '@mui/material/Typography'
 import { styled } from '@mui/material/styles'
 import { COLORS } from '../../../../../styles/theme/colors'
 import { getNetworkNames, Network } from '../../../../../types/network'
-import Collapse from '@mui/material/Collapse'
 import { RouteUtils } from '../../../../utils/route-utils'
+import {
+  Select,
+  SelectOptionBase,
+  StyledSelectButton,
+  StyledSelectListbox,
+  StyledSelectOption,
+} from '../../../../components/Select'
+import { SelectRootSlotProps } from '@mui/base/Select'
+import ExpandLessIcon from '@mui/icons-material/ExpandLess'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { getNetworkIcons } from '../../../../utils/content'
+import { optionClasses } from '@mui/base/Option'
 
-const StyledNetworkSelector = styled(Box)(() => ({
+interface NetworkOption extends SelectOptionBase {
+  label: Network
+  value: Network
+}
+
+const StyledNetworkSelector = styled(Select<NetworkOption>)(({ theme }) => ({
   position: 'absolute',
-  bottom: 0,
+  bottom: theme.spacing(5),
   display: 'flex',
   width: '100%',
   justifyContent: 'center',
 }))
 
-const StyledBox = styled(Box)(({ theme }) => ({
+const StyledButton = styled(StyledSelectButton)(({ theme }) => ({
+  height: '44px',
+  minWidth: '200px',
   backgroundColor: theme.palette.layout.primaryBackground,
-  border: `solid 3px ${theme.palette.layout.lightBorder}`,
-  borderRadius: '45px',
-  padding: theme.spacing(3, 4),
-  display: 'inline-flex',
-  alignItems: 'center',
+  border: `2px solid ${theme.palette.layout.lightBorder}`,
+  color: theme.palette.layout.main,
+  '&:hover': {
+    backgroundColor: theme.palette.layout.primaryBackground,
+  },
 }))
 
-export const SelectNetworkButton = styled(Button, {
-  shouldForwardProp: prop => prop !== 'isSelectedNetwork',
-})<{ isSelectedNetwork: boolean }>(({ isSelectedNetwork, theme }) => ({
-  height: '30px',
-  padding: theme.spacing(2, 3),
-  textTransform: 'capitalize',
-  fontSize: '16px',
-  borderRadius: '9px',
-  backgroundColor: isSelectedNetwork
-    ? theme.palette.layout.primaryBackground
-    : theme.palette.layout.secondary,
-  borderColor: isSelectedNetwork ? theme.palette.layout.hoverBorder : theme.palette.layout.secondary,
-  borderWidth: theme.spacing(1),
-  color: theme.palette.layout.main,
-  '&:hover, &:focus-visible': {
-    backgroundColor: theme.palette.layout.secondary,
-    borderWidth: theme.spacing(1),
-    borderColor: COLORS.white,
+const StyledListbox = styled(StyledSelectListbox)(() => ({
+  minWidth: '200px',
+  background: COLORS.white,
+  color: COLORS.grayDark,
+}))
+
+const StyledOption = styled(StyledSelectOption)(() => ({
+  height: '44px',
+  color: COLORS.grayDark,
+  svg: {
+    color: COLORS.grayExtraDark,
+  },
+  [`&:hover:not(.${optionClasses.disabled}),
+  &.${optionClasses.highlighted}`]: {
+    backgroundColor: 'transparent',
```

Comment:
Did you mean to give it some color when hovered / focused?
```suggestion
  [`&:hover:not(.${optionClasses.disabled}),
  &:focus-visible:not(.${optionClasses.disabled}),
  &.${optionClasses.highlighted}`]: {
    backgroundColor: 'red',
```

Reply:
Right, thanks for noticing.